engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformulamulticolumn/RollingFormulaMultiColumnOperator.java

```diff
@@ -409,47 +404,39 @@ protected static IntConsumer getChunkNullSetter(final WritableChunk<? extends Va
      * particularly efficient but is only be called once per bucket.
      */
     private static void assignKeyValue(
-            @NotNull final Chunk<?> valueChunk,
-            final int index,
-            @NotNull SingleValueColumnSource<?> inputColumnSource) {
-        final ChunkType chunkType = valueChunk.getChunkType();
+            @NotNull final Object keyValue,
+            @NotNull final SingleValueColumnSource<?> inputColumnSource) {
+
+        final ChunkType chunkType = inputColumnSource.getChunkType();
```

Comment:
You probably don't need any of this. WritableColumnSource allows you to set with a boxed value. I believe you are working with outer (not reinterpreted values) throughout the stack, here.